src/components/Lend/AaveAction.tsx

```diff
@@ -21,7 +21,7 @@ export const deposit = async (provider, accountAddress) => {
     // reserve: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
     // DAI reserve polygon
     reserve: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
-    amount: "10",
+    amount: "1",
```

Comment:
not needed?

Reply:
want to push your changes first? and then I can just discard this... I edited this b/c I didn't know if I had enough DAI in my Torus wallet for the tx to go through.

Reply:
makes sense, though in this case we don't need to commit the change right? just a value for testing locally

Reply:
yup your recent PR should also have this change.

Reply:
something to discuss actually -- do we want the user to enter in an amount to supply, or automatically supply everything

Reply:
i like the former....